extension/lib/settings_wrapper.js

```diff
@@ -78,15 +78,29 @@ class Settings {
 		});
 	}
 
-	/*
-	Save the term document frequencies. 
-	This is a map of all words user has ever read to the number of documents they have occured in. 
-	Every time they read a new document, it adds one to each word in the dict, or adds the word to the dict
-	initialized at 1. 
-		- trackerSettings: {trackerSettingKey : termDocumentFreq}
-		Note termDocumentFreq is a dict. 
-	*/
-	// TODO: Build in safeguard against exceeding storage limit
+	// FOR DEV ONLY; REMOVE BEFORE PRODUCTION
+	// resetTermDocumentFreq() {
+	// 	let termDocumentFreq = {};
+	// 	chrome.storage.local.set({[settingKey.TERM_DOCUMENT_FREQ]: termDocumentFreq}, function() {
+	// 		if (chrome.runtime.lastError) {
+	// 			console.log("Failed to save term document frequencies: " + chrome.runtime.lastError);
+	// 			return;
+	// 		}
+	// 	});
+	// }
+
+	// FOR DEV ONLY; REMOVE BEFORE PRODUCTION
+	// resetVisitedUrls() {
+	// 	let visitedUrls = {};
+	// 	chrome.storage.local.set({[settingKey.VISITED_URLS]: visitedUrls}, function() {
+	// 		if (chrome.runtime.lastError) {
+	// 			console.log("Failed to save term document frequencies: " + chrome.runtime.lastError);
+	// 			return;
+	// 		}
+	// 	});
+	// }
```

Comment:
Until TF-IDF is fully up, these functions are useful for dev purposes. Will remove before production. They clear out the settings. We might also consider having them as backup options in case a user screws up their dictionaries. In this case we could replace the empty {} values with a hard-coded preset.